ld_ex1.py

```diff
@@ -0,0 +1,10 @@
+#program to sum all elements in a list
+if __name__ == "__main__":
+    lst = [] #defining empty list which contain the elements the user will enter, then we want to sum all elements in this list.
+    x = int(input("Please enter number of elements: ")) #total number of elements in our cute list
+
+for i in range(x): #we want to insert an element for each index in the list. if x equal to 2, we want the user to insert 2 elemnts.
+    e = int(input("Enter element:")) #we want that each element will be a type of integer
+    lst.append(e) #adding each element (e) to the our list (lst)
+
+print("Sum of elements in the given list is: ", sum(lst)) #required output
```

Comment:
1.  You are missing a whole TAB on lines 6-10, fix it, these lines are outside of your MAIN!
2. I love that you document on every line but it is unnecessary, it turns the code very messy, try documenting only where the code is more complicated, places where you might think that it would be hard to understand what you did just but reading. 
for example, in line 3, you create a list, every Python programmer knows this syntax so there is no need explaining. if i were you, I would only state its purpose, for say: #List for holding received elements

Approved.